pkg/jobrunaggregator/jobrunaggregatorlib/disruption.go

```diff
@@ -48,6 +49,44 @@ type BackendDisruption struct {
 	DisruptionMessages []string
 }
 
+// defined in origin/platformidentification/types.go
+// minimized here
+type ClusterData struct {
+	MasterNodesUpdated string
+}
+
+func GetMasterNodesUpdatedStatusFromClusterData(clusterData map[string]string) string {
```

Comment:
Can you godoc the a little here, i'm confused about why clusterData is a map of string to string. I'd expect we were working with a single cluster data file per job run here. Is this because GetOpenShiftTestsFilesWithPrefix was built to potentially return multiple files, but in this case we'd expect 1 or 0?

Reply:
Added, I also had a comment within the function covering the case.  It is because like disruption data a copy is created for both the upgrade and conformance portions of the job runs.  We can reevaluate that if we want.

Reply:
Aha I see now, thanks!